src/backend/gl/Cargo.toml

```diff
@@ -30,7 +30,7 @@ parking_lot = "0.11"
 raw-window-handle = "0.3"
 
 [target.'cfg(not(target_arch = "wasm32"))'.dependencies]
-egl = { package = "khronos-egl", version = "4", features = ["dynamic"] }
+egl = { package = "khronos-egl", version = "4.1", features = ["dynamic"] }
```

Comment:
Includes fix https://github.com/timothee-haudebourg/khronos-egl/pull/19

Without this patch we can't use `gfx-backend-gl` on Android devices.